reporter/src/main/kotlin/reporters/ctrlx/CtrlXAutomationReporter.kt

```diff
@@ -62,12 +64,21 @@ class CtrlXAutomationReporter : Reporter {
                 License(name = id, spdx = it, text = text.orEmpty())
             }
 
+            // The specification requires at least one license.
+            val componentLicenses = licenses.takeUnless { it.isNullOrEmpty() } ?: listOf(
```

Comment:
Maybe this list with a dummy license can be extracted into a constant with a speaking name.

Reply:
Done: https://github.com/oss-review-toolkit/ort/pull/4769